text/????-separate-semantic-conventions.md

```diff
@@ -0,0 +1,115 @@
+# Separate Semantic Conventions
+
+Move Semantic Conventions outside of the main Specificaiton and version them
+separately.
+
+## Motivation
+
+We need to allow semantic conventions to evolve independently of the overall
+OpenTelemetry specification. Today, any breaking change in a semantic convention
+would require bumping the version number of the entirety of the OpenTelemetry
+specification.
+
+## Explanation
+
+A new github repository called `semantic-conventions` would be created in the
+OpenTelemetry organization.
+
+This would have the following structure:
+
+- Boilerplate files, e.g. `README.md`, `LICENSE`, `CODEOWNERS`, `CONTRIBUTING.md`
+- `Makefile` that allows automatic generation of documentation from model.
+- `model/` The set of YAML files that exist in
+  `{specification}/semantic_conventions` today.
+- `docs/` A new directory that contains human readable documentation for how to
+  create instrumentation compliant with semantic conventions.
+  - `resource/` - Contents of `{specification}/resource/semantic_conventions`
+  - `trace/` - Contents of `{specification}/trace/semantic_conventions`
```

Comment:
We have already discussed abandoning the trace/metric/log folder structure, and replacing it with a folder per domain. This structure would keep all of the relevant information grouped together for anyone attempting to instrument their code.

Reply:
I think there is a merit to keeping the scope smaller here as well.

Reply:
Added callouts to this as future direction.